qiskit/transpiler/preset_passmanagers/builtin_plugins.py

```diff
@@ -468,7 +483,21 @@ def _opt_control(property_set):
                     Optimize1qGatesDecomposition(
                         basis=pass_manager_config.basis_gates, target=pass_manager_config.target
                     ),
-                    CXCancellation(),
+                    InverseCancellation(
+                        [
+                            CXGate(),
+                            ECRGate(),
+                            CZGate(),
+                            CYGate(),
+                            XGate(),
+                            YGate(),
+                            ZGate(),
+                            HGate(),
+                            SwapGate(),
+                            (TGate(), TdgGate()),
+                            (SGate(), SdgGate()),
+                        ]
+                    ),
```

Comment:
`(SXGate(), SXdgGate())` could be eligible for inclusion as well, and `sx` turns up more commonly in backends we decompose for including Clifford decompositions.

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/11210/commits/7b99040881cc5cb30ccfe58b69f7119da8d05708